### PR TITLE
Feature/background image and license

### PR DESCRIPTION
**Pull request by lukasz-bien-rmtcfm-com**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,20 @@
 {
     "name": "redis-rate-limiting",
-    "version": "1.0.0",
-    "description": "",
+    "version": "0.0.1",
+    "description": "Redis request rate limit demo",
+    "keywords": [
+        "NodeJS",
+        "ExpressJS",
+        "Redis"
```
Comment (rafal-ksiazek-rmtcfm-com) [inline package.json:8]: add "Rate limit"

File: package.json [hunk 2]
```diff
@@ -1,16 +1,20 @@
 {
     "name": "redis-rate-limiting",
-    "version": "1.0.0",
-    "description": "",
+    "version": "0.0.1",
+    "description": "Redis request rate limit demo",
```
Comment (rafal-ksiazek-rmtcfm-com) [inline package.json:4]: requests - (s) at the end

File: src/public/styles.css [hunk 3]
```diff
@@ -13,3 +13,12 @@
 #limit-select {
     max-width: 280px;
 }
+
+@media screen and (min-width: 680px) {
+    body {
+        background-image: url('RedisLabs_Illustration.svg');
```
Comment (rafal-ksiazek-rmtcfm-com) [inline src/public/styles.css:19]: I think all urls should be started with './' or even '/'

File: package.json (hunk 1 above)
Comment (lukasz-bien-rmtcfm-com) [reply package.json:8]: done

File: package.json (hunk 2 above)
Comment (lukasz-bien-rmtcfm-com) [reply package.json:4]: done

File: src/public/styles.css (hunk 3 above)
Comment (lukasz-bien-rmtcfm-com) [reply src/public/styles.css:19]: done

**Pull request merged**